damnit/backend/listener.py

```diff
@@ -145,13 +106,27 @@ def handle_event(self, record, msg: dict, run_data: RunData):
         log.info("Processing output will be written to %s",
                  log_path.relative_to(self.context_dir.absolute()))
 
-        with log_path.open('ab') as logf:
-            # Create subprocess to process the run
-            extract_proc = subprocess.Popen([
-                sys.executable, '-m', 'damnit.backend.extract_data',
-                str(proposal), str(run), run_data.value
-            ], cwd=self.context_dir, stdout=logf, stderr=subprocess.STDOUT)
-        self.extract_procs_queue.put((proposal, run, extract_proc))
+        python_cmd = [
+            sys.executable, '-m', 'damnit.backend.extract_data',
+            str(proposal), str(run), run_data.value
+        ]
+
+        res = subprocess.run([
+            'sbatch', '--parsable',
+            '--cluster=solaris',
+            '-o', log_path,
+            # Default 4 CPU cores & 25 GB memory, can be overridden
+            '--cpus-per-task', str(self.db.metameta.get('noncluster_cpus', '4')),
+            '--mem', self.db.metameta.get('noncluster_mem', '25G'),
```

Comment:
These options should be in the documentation. Also not a fan of the names, but I can't think of anything better so let's roll with it :shrug:

Reply:
Thanks, I've described them (along with the default limits) in the docs.